### ISSUE TITLE
Emulation of SC-3000

### ISSUE THREAD

**Issue by bodhi-baum** [drhelius/Gearsystem#92]
Would it be possible to implement emulation of the SC-3000? 

In fact it seems to be a lot like the SG-1000 but has some enhancements like SF-7000 and Tape-recording.

**Comment by drhelius** [comment]
You are right it’s almost identical to SG-1000 but I’m not really interested in emulating computers right now.
I’m not going to work on it in the short term but I’ll keep the issue for the future.
